apps/docs/src/common/pages/get-started/start-prototype.tsx

```diff
@@ -10,7 +10,7 @@ const Page: FC<PageProps> = ({ location }) => (
   <div className="govuk-grid-row">
     <Helmet>
       <title>Guidance on Home Office Design System</title>
-      <meta name="og:article:section" content="Get started" />
+      <meta name="og:article:section" content="Starting a new prototype" />
```

Comment:
This was actually correct to begin with. - It's the section that the page sits in.

Reply:
The bit that needed to change was the `export const` line at the bottom.

Reply:
sorry, I've just amended this